### ISSUE TITLE
Update package for usage with rollup >= 1

### ISSUE THREAD

**Issue by vladshcherbin** [MikeKovarik/rollup-plugin-notify#5]
Hey,

I'm using rollup 1.2.1 and while the package works, it gives a note:

```
'rollup-plugin-notify' will not work. Rollup 0.60.0 or higher is required
```

I see a check for minor version (<60) and in version 1.2.* it's equal to 2. Would be nice to have an update for current rollup version.

Also, `success` option is missing in the docs.

**Comment by MikeKovarik** [comment]
fixed with #4

**Comment by vladshcherbin** [comment]
@MikeKovarik thanks 😉
